socialhome/streams/app/stores/streamStore.js

```diff
@@ -1,31 +1,52 @@
 import Vue from "vue"
 import Vuex from "vuex"
-import isNil from "lodash/isNil"
+import ReconnectingWebSocket from "reconnecting-websocket"
 
-// Vue setup
+import getState from "streams/app/stores/streamStore.state"
+import {actions, mutations, stateOperations} from "streams/app/stores/streamStore.operations"
+
+/* This should actually rely in ``main.js``.
```

Comment:
I think all the plugin initialisation should be in only one place. With that problem, we are forced to put the `Vue.use(Vuex)` statement in every store as we can't guaranty in which order the statement are going to be generated. It's bloated.